### DISCUSSION TITLE
Uploading with chunks

### DISCUSSION THREAD

**Discussion by Yoosk** [SergiX44/XBackBone#360] [Ideas]
**Is your feature request related to a problem? Please describe.**
It's related to cloudflare limit of maximum 100mb for file uploading. 

**Describe the solution you'd like**
As far as i am aware, sharex does allow uploading in chunks, which would give possibility to upload files over 100mb even when using cloudflare DNS, like i currently do.

**Additional context**
Example from another uploader im currently using 
![2020-06-29_17-35-18-sublime_text](https://user-images.githubusercontent.com/4502802/86025987-2ed28800-ba2f-11ea-827c-b934b877750f.png)
which is https://github.com/BobbyWibowo/lolisafe

**Comment by sergix44** [comment]
I already looked into that when talking about resumable file uploads:
https://github.com/SergiX44/XBackBone/issues/98

The main issue is about the remote storage (GCP, Amazon S3, Dropbox...) drivers, as the chunks should be stored locally on the server, then after the upload is completed recompose the original file and finally upload to the remote storage service. So as you can imagine it cannot be fast as "redirect" the upload stream as is to the remote storage service. So I don't know if is worth it in this case.

I'm open to suggestions.

**Comment by andrew121410** [comment]
Oh so this is the issue I'm currently facing then my website is proxied through cloudflare and xbackbone isn't working with uploading anything above 100MB.

Nextcloud seems to work though. damn

**Comment by sergix44** [comment]
Another issue is even when you may be able to store the file chunks and than glue them together, basically any file uploader (sharex, screencloud, etc) is NOT support this upload method, they only throw a single POST request to the endpoint, so you may be able to upload bigger file from the webui, but any client will basically not work.
